src/hotspot/share/utilities/debug.cpp

```diff
@@ -491,16 +489,16 @@ extern "C" void pp(void* p) {
 
 
 // pv: print vm-printable object
-extern "C" void pa(intptr_t p)   { ((AllocatedObj*) p)->print(); }
-extern "C" void findpc(intptr_t x);
-
-#endif // !PRODUCT
+#ifndef PRODUCT
+// AllocatedObj in allocation.hpp is not defined for PRODUCT
+extern "C" JNIEXPORT void pa(intptr_t p)   { ((AllocatedObj*) p)->print(); }
+#endif
```

Comment:
I don't really think 'pa' is interesting enough to keep.

Reply:
Thanks Coleen - I haven't used pa either.  Will update without it...

Reply:
I've taken out pa(intptr_t) and the misleading comment above it saying "pv".  
Also removed pv() from the help, which was perhaps an old name for pa.